framework/draw/internal/fontfaceft.cpp

```
@@ -573,6 +573,44 @@ f26dot6_t FontFaceFT::capHeight() const
     return gm->bbox.height();
 }
 
+f26dot6_t FontFaceFT::underlinePosition() const
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)